source/FreeRTOS_Sockets.c

```diff
@@ -4098,7 +4101,7 @@ void vSocketWakeUpUser( FreeRTOS_Socket_t * pxSocket )
             size_t uxLittlePerc = sock20_PERCENT;
             size_t uxEnoughPerc = sock80_PERCENT;
             size_t uxSegmentCount = pxSocket->u.xTCP.uxRxStreamSize / pxSocket->u.xTCP.usMSS;
-            static const struct xPercTable
+            static const struct Percent
```

Comment:
Why was this structure name changed?

We are not changing that below in line 4108?

Reply:
This is the datatype name, below is the variable.